src/PrestaShopBundle/Resources/views/Admin/Module/Includes/action_button.html.twig

```diff
@@ -23,7 +23,7 @@
  * International Registered Trademark & Property of PrestaShop SA
  *#}
 
-{% set displayAction = action|capitalize|replace({'_': " "})|trans({}, 'Admin.Actions') %}
+{% set displayAction = action|title|replace({'_': " "})|trans({}, transDomain[action]) %}
 
```

Comment:
the **capitalize** filter would only capitalize the first character, here we need a **ucwords** kind of filter. Happily the **title** filter does just that.

Reply:
Shouldn't the `trans` filter be called before using the `title` one? Or else the key might not be recognized?

Reply:
Good question @jolelievre, initially the **capitalize** filter was used to build the translation key. But it failed, because capitalize makes a **ucfirst**, when what was needed to build a proper translation key was a **ucwords** (in this case anyway). The title filter does the needed **ucwords**

Reply:
Ok! then you need top check if the keys are present in the catalogue. Because the catalog is built by scanning the PrestaShop code and fetching all the hard coded strings. It is not able to compute variables. So unless these keys are used explicitly somewhere else in the code they won't be present in the catalogue and won't be translated.

Reply:
Yes, they exist in the catalog, from what I've seen and understood, those values are used as constants in the code (php side), and made to be transformed into translation keys easily (twig side).